source/common/stats/raw_stat_data.cc

```diff
@@ -22,6 +22,8 @@ uint64_t roundUpMultipleNaturalAlignment(uint64_t val) {
   return (val + multiple - 1) & ~(multiple - 1);
 }
 
+const uint64_t MaxNameLength = 255;
```

Comment:
I'm pretty sure all `RawStatData` stuff is no longer used. I think all of this can be completely deleted. At this point we only have the heap allocator.

Reply:
Agreed, but it sounds like that particular cleanup might be left to @jmarantz.

Reply:
See comment above. Please delete. This code is dead.

Reply:
Done.